app/src/main/java/com/github/se/studybuddies/ui/topics/TopicScreen.kt

```diff
@@ -406,12 +434,21 @@ fun AddInFolderButton(
     uid: String,
     folderFieldVisible: MutableState<Boolean>,
     fileFieldVisible: MutableState<Boolean>,
-    parentUID: MutableState<String>
+    childrenExpanded: MutableState<Boolean>,
+    parentUID: MutableState<String>,
+    depth: Int
 ) {
   val expandedState = remember { mutableStateOf(false) }
-  Box(modifier = Modifier.fillMaxWidth()) {
+  val screenWidth = LocalConfiguration.current.screenWidthDp
+  val offset =
+      if (depth > 0) DpOffset((screenWidth - 180).dp, 0.dp) else DpOffset(x = (-16).dp, y = 0.dp)
```

Comment:
In the first DpOffset, you didn't give the parameter value is assigned to, but in the second yes (x = ..., y= ...). Try to stay consistent between the two (either but for both, or for none)

Additionnaly you could make the line more readable by not calling DpOffset twice for only one difference.
You could do this instead : `val offset = DpOffset((if (depth > 0) (screenWidth - 180) else (-16)).dp, 0.dp)` but it still no very easy to read, so I recommend to extract it into a new value :
```
val xOffset = if (depth > 0) (screenWidth - 180) else (-16)
val offset = DpOffset(xOffset.dp, 0.dp)
```